lib/sla.js

```diff
@@ -1,6 +1,77 @@
-'use strict';
+/*jslint node: true */
+'use strict'
 
-module.exports = function(callback){
-    console.log("\"snarflegluff\" should be printed below:");
-    var str = "snarflegluff";
-    return str;}
+/**
+ * Looks for an existing mixing bridge and creates one if none exist.
+ */
+function designateMixingBridge(client, callback) {
+  client.bridges.list(function(err, bridges) {
+    var bridge = bridges.filter(function(candidate) {
+      return candidate.bridge_type === 'mixing';
```

Comment:
Is this the expected behavior? You never check to see if the bridge has channel(s) in it and since it's a mixing bridge, the new channel will hear whatever other channels are currently hearing and vice versa.